.git-blame-ignore-revs

```diff
@@ -1,3 +1,6 @@
 # enforce unified code style with black
 28914fc2236cd5850f536b151d0ef383855d8519
 db0691cfb6b033e55664ffba25deb456a386a48c
+
+# enforce unified code style in our documentation with blackdoc
+a26a52d037a265774be0ca06bb141e3346e73ad0
```

Comment:
I think the conclusion in https://github.com/pyvista/pyvista/pull/4129 was to ignore that too (basically one PR broken in two due to a late reconfiguration of blackdoc):
```suggestion
a26a52d037a265774be0ca06bb141e3346e73ad0
635052d1bce8615fcc4879f5f200921a4f78aa1a
```
Or did I misunderstand?

Reply:
You're correct. Thanks for catching that!